tests/check/check.go

```diff
@@ -72,12 +72,14 @@ func runTests(t *testing.T, schemaVersion string, client CheckTestClientInterfac
 	ctx := context.Background()
 
 	for _, test := range testCases.Tests {
-		resp, err := client.CreateStore(ctx, &openfgapb.CreateStoreRequest{Name: test.Name})
-		require.NoError(t, err)
+		test := test
+		t.Run(test.Name, func(t *testing.T) {
+			t.Parallel()
```

Comment:
There's something odd going on. This doesn't seem to be parallelizing them.  Or, something else (in the setup?) is taking a very long time

<img width="514" alt="image" src="https://user-images.githubusercontent.com/5374887/222223401-cd572e61-bc35-44c2-a2b1-13226026f480.png">

Reply:
That may just be the startup process for the MySQL container

Reply:
I think it's that, but it's odd because I see the container pop-up in the UI very quickly. Maybe the tear down is also slow?

Reply:
In any case, I see an improvement 

<img width="1919" alt="main" src="https://user-images.githubusercontent.com/5374887/222228054-a7864e35-c10c-479c-9b52-e139bde8afc9.png">

<img width="1919" alt="thisPR" src="https://user-images.githubusercontent.com/5374887/222228083-3eb9245a-1f59-4412-adb3-14b1004c7453.png">